### PR TITLE
Content/seamless mobile app experience blog post

### PR DESCRIPTION
**Pull request by fuentespatrick**
Add blog post for improved mobile app experience

### CODE REVIEW

**Review by nohe427 — DISMISSED** [review]

File: pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+!!! aside.message--note
+**Note:** This window management feature applies for apps installed via Play Store, not adb (Android Debug Bridge).
+!!!
+
+## Adapt your Android apps for Chrome OS
+
+Better window management for Android mobile apps is an important step toward creating a seamless user experience on larger screens. But the best way to make sure users get the most out of your app on Chrome OS is to adapt it for desktop use.
+
+We broke down some [helpful steps](https://chromeos.dev/en/posts/optimizing-android-app-experiences-for-chromeos) to ensure your app looks and works great across form factors and screen sizes, as well as some [tips for optimizing for Chrome OS.](/{{locale.code}}/android)
```
Comment (joycetoh8) [inline pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md:59]:
```suggestion
We broke down some [helpful steps](/{{locale.code}}}/posts/optimizing-android-app-experiences-for-chromeos) to ensure your app looks and works great across form factors and screen sizes, as well as some [tips for optimizing for Chrome OS.](/{{locale.code}}/android)
```

File: pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Users can easily [switch between a predefined phone and tablet view](https://support.google.com/chromebook/thread/124854535/introducing-chrome-os-93-to-the-stable-channel?hl=en) by using a simple window resize UI that appears when they hover over the top of the app screen. Users can also disable this window management feature to enable full resizability by clicking the “Resizable” option.
+
+![](/images/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os/window-management.jpg)
+
+This feature applies to most apps available in the Play Store not currently optimized for desktop use. Games and Media apps that have fullscreen mode enabled to [deliver more immersive experiences](https://developer.android.com/training/system-ui/immersive) will continue to launch in fullscreen mode, but users can change the window view at any time.
+
+For more details and tips, check out our post about [optimizing your Android apps for desktops.](/{{locale.code}}/android)
```
Comment (joycetoh8) [inline pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md:38]: Is this the right link?

Comment (fuentespatrick) [reply pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md:38]: Yeah, that's what is in the article draft. Thanks for checking!

File: pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md (hunk 1 above)
Comment (fuentespatrick) [reply pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md:59]: Good catch! I appreciate the attention to detail!

**Review by joycetoh8 — DISMISSED** [review]

**Comment by Psoffie009** [comment]
> Add blog post for improved mobile app experience

**Pull request merged**